sklearn/utils/validation.py

```diff
@@ -910,10 +904,10 @@ def check_is_fitted(estimator, attributes, msg=None, all_or_any=all):
     if not hasattr(estimator, 'fit'):
         raise TypeError("%s is not an estimator instance." % (estimator))
 
-    if not isinstance(attributes, (list, tuple)):
-        attributes = [attributes]
+    attrs = [v for v in vars(estimator) if v.endswith("_")
```

Comment:
I think NearedtNeighbors has stored only _fit_X

Reply:
It has this:

https://github.com/scikit-learn/scikit-learn/blob/7c60ead7936027d4f2515b61abdac4e0288e4074/sklearn/neighbors/base.py#L166-L169

and it was very recently documented.

Reply:
Common tests pass so it must work ;)